web-api/terraform/template/main.tf

```diff
@@ -0,0 +1,33 @@
+module "api-east" {
+  source         = "../api/"
+  environment    = var.environment
+  dns_domain     = var.dns_domain
+  authorizer_uri = aws_lambda_function.cognito_authorizer_lambda.invoke_arn
+  account_id     = data.aws_caller_identity.current.account_id
+  zone_id        = data.aws_route53_zone.zone.id
+  lambda_environment = merge(data.null_data_source.locals.outputs, {
+    DYNAMODB_ENDPOINT = "dynamodb.us-east-1.amazonaws.com"
+  })
+  region   = "us-east-1"
+  validate = 1
+  providers = {
+    aws = aws.us-east-1
+  }
+}
+
+module "api-west" {
+  source         = "../api/"
+  environment    = var.environment
+  dns_domain     = var.dns_domain
+  authorizer_uri = aws_lambda_function.cognito_authorizer_lambda.invoke_arn
+  account_id     = data.aws_caller_identity.current.account_id
+  zone_id        = data.aws_route53_zone.zone.id
+  lambda_environment = merge(data.null_data_source.locals.outputs, {
+    DYNAMODB_ENDPOINT = "dynamodb.us-west-1.amazonaws.com"
+  })
+  region   = "us-west-1"
+  validate = 0
```

Comment:
Why set this to false?

Reply:
I'm unsure the reason here, but, in the interest of moving the PR along while the author is away, I can create a Trello card to come back and address this (if it needs addressing). Since west is not used for as many of the same services as east, it could be intentional.

Reply:
Digging in a little further, this appears to be a flag that sets the `count` on the aws cert validation and route 53 terraform configs. (see `api-public.tf`, `websockets.tf`, `api.tf`) - and appears to be intentional.

Reply:
Thanks!